src/test/java/com/conveyal/gtfs/graphql/GTFSGraphQLTest.java

```diff
@@ -164,6 +173,30 @@ public void canFetchCalendars() {
         });
     }
 
+    /** Tests that the location groups of a feed can be fetched. */
+    @Test
+    public void canFetchLocationGroups() {
+        assertTimeout(Duration.ofMillis(TEST_TIMEOUT), () -> {
```

Comment:
Ok I didn't realize the pattern in this file. All these can-do tests should become one single parametrized test that accepts an input file (and a description, optionally) and this file will be much shorter and easier to read.

Reply:
Unfortunately the method name is used to lookup the matching JSON output, so I've left as is.